OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectWorld.cs

```diff
@@ -100,12 +124,34 @@ sealed class DreamMetaObjectWorld : IDreamMetaObject {
                     return new DreamValue(_dreamMapManager.Size.Y);
                 case "maxz":
                     return new DreamValue(_dreamMapManager.Levels);
-                //case "address":
-                //    return new(Runtime.Server.Address.ToString());
-                //case "port":
-                //    return new(Runtime.Server.Port);
-                //case "url":
-                //    return new("opendream://" + Runtime.Server.Address + ":" + Runtime.Server.Port);
+                case "address": // By address they mean, the local address we have on the network, not on the internet.
+                    var host = Dns.GetHostEntry(Dns.GetHostName());
+                    var ip_type = AddressFamily.InterNetwork;
+                    if (Display_IPv6)
+                    {
+                        ip_type = AddressFamily.InterNetworkV6;
+                    }
+                    foreach (var ip in host.AddressList)
+                    {
+                        if (ip.AddressFamily == ip_type)
```

Comment:
This can be shortened to a ternary, and should be renamed to `ipType` to fit the naming convention of other local variables
```suggestion
                    var ipType = DisplayIPv6 ? AddressFamily.InterNetworkV6 : AddressFamily.InterNetwork;

                    foreach (var ip in host.AddressList)
                    {
                        if (ip.AddressFamily == ipType)
```